lib/vrt/node.rb

```diff
@@ -20,7 +20,15 @@ def children?
     end
 
     def construct_vrt_id
-      parent ? "#{parent.qualified_vrt_id}.#{id}" : id.to_s
+      id_list.join('.')
+    end
+
+    def cvss_v3
```

Comment:
how do you feel about dynamically defining the method for each mapping definition?

Reply:
I mean, we could. I thought about it too. But I was thinking of waiting until we have a second mapping before we go that road. Only because having a whole meta programming thing to loop though a list and define dynamic methods just for a single method seemed overkill.